guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -173,3 +173,22 @@ endif::[]
 . Click *Generate*.
 . Copy the generated `curl` command.
 . On the host that you want to register, run the `curl` command as `root`.
+
+.CLI procedure
+
+. Use the `hammer host-registration generate-command` to generate the `curl` command to register the host.
+. On the host that you want to register, run the `curl` command as `root`.
+
+For more information, enter the `hammer host-registration generate-command --help` command.
```

Comment:
Could this be
```suggestion
For more information, see the CLI help with `hammer host-registration generate-command --help`.
```
(or "with the `…` command")?

Reply:
(What I dislike is "enter command" without telling what the command will do -- even if it's rather obvious from the `--help` part)